packages/dm-core-plugins/src/attribute-selector/AttributeSelectorPlugin.tsx

```diff
@@ -26,14 +26,16 @@ export type TAttributeSelectorPluginConfig = {
   childTabsOnRender?: boolean
   homeRecipe?: string
   asSidebar?: boolean
+  visibleAttributes?: string[]
 }
 
 export const AttributeSelectorPlugin = (props: IUIPlugin): JSX.Element => {
   const { idReference, config: passedConfig, onSubmit } = props
   const config: TAttributeSelectorPluginConfig = {
-    childTabsOnRender: passedConfig?.childTabsOnRender ?? false,
+    childTabsOnRender: passedConfig?.childTabsOnRender ?? true,
```

Comment:
I agree with the change, but I think you should also update the default value in the AttributeSelectorPluginConfig files

Reply:
fixed

Reply:
You forgot the one in the example folder (uppermost file in this PR)